xsl/localizations/fr-CA.xml

```diff
@@ -148,6 +165,8 @@ along with PreTeXt.  If not, see <http://www.gnu.org/licenses/>.
     <localization string-id='hint'>Indice</localization>  <!-- Knowingly differs from fr-FR, which uses "Indication" instead -->
     <localization string-id='answer'>Réponse</localization>
     <localization string-id='solution'>Solution</localization>
+    <!-- A group of divisional exercises (with introduction and conclusion) -->
+    <localization string-id='exercisegroup'>Groupe d'exercices</localization>
```

Comment:
@rbeezer should weigh in, but maybe a bare apostrophe could cause trouble. Should it use `<rsq/>`?

Reply:
Not fast enough.  ;-)  I think a bare apostrophe is fine, authors should be doin' that all the time.  Unclear the exact utility of <rsq/>.  ;-)

Reply:
It would be for future proofing, since `'` can be a special character. Just a hair more likely to evade the escaping that we do, since this is an automatically generated string, not author input.

Reply:
Better safe than sorry they say